idb/postgres/postgres.go

```diff
@@ -1719,6 +1720,69 @@ func (db *IndexerDb) GetAccounts(ctx context.Context, opts idb.AccountQueryOptio
 	return out, round
 }
 
+func (db *IndexerDb) checkAccountResourceLimit(ctx context.Context, tx pgx.Tx, opts idb.AccountQueryOptions) error {
```

Comment:
You can skip the checks if all of the include flags are already false.

Reply:
thanks, updated